build.gradle

```diff
@@ -1,21 +1,24 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    apply from: 'versions.gradle'
+    addRepos(repositories)
```

Comment:
Following `repositories {...}` is replaced by `addRepos(repositories)` so should be no longer needed.